include/adm/private/changed_id_traits.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef LIBADM_CHANGED_ID_TRAITS_HPP
+#define LIBADM_CHANGED_ID_TRAITS_HPP
```

Comment:
should use pragma once

Reply:
fbace6b89c0c5e2ae017d5ca02d79ee4fc76f39b